src/BreadCrumbs.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+const crumbs = ['Home', 'StudentList', 'DetailResult'];
+
+export default class BreadCrumbs extends React.Component {
+  breadCrumbClick(page) {
+    return this.props.breadCrumbClick(page);
+  }
+
+  render() {
+    let fCrumbs = [...crumbs];
+    this.props.currentPage != 'DetailResult' ? fCrumbs = crumbs.slice(0, 2) : fCrumbs = crumbs;
```

Comment:
fCrumbs = this.props.currentPage != 'DetailResult' ?  crumbs.slice(0, 2) : crumbs;

Reply:
resolved in eslint correction.